be/src/models/user.model.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, Column, PrimaryGeneratedColumn, Unique, BeforeInsert, BeforeUpdate } from 'typeorm';
+import { ObjectType, Field } from '@nestjs/graphql';
+
+
+@ObjectType()
+@Entity('users')
+export class UserModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Field()
+  @Column({ length: 255, nullable: false })
+  name: string;
+  @Column({ length: 255, nullable: false})
+  public password: string;
```

Comment:
why is this public? (genuine question, nut sure if should be)

Reply:
ah left over from some tests if i could get crypting to work.